server/src/server/session/HadoopGraphFactory.java

```diff
@@ -39,10 +39,13 @@
  */
 public class HadoopGraphFactory {
 
-    private Config config;
+    protected Config config;
```

Comment:
is it protected for use by KGMS? If yes let's add this comment so the next person won't change it to `private`:
```
// NOTE: this field is used by Grakn KGMS and should be kept as 'protected'
```